documents/credential-branding.md

```diff
@@ -0,0 +1,87 @@
+
+    - **Logo (OPTIONAL):** A JSON object with information about the logo of the credential. This object may include:
+        - **URL (OPTIONAL):** The URL where the wallet or verifier can obtain the credential's logo from the credential issuer.
+        - **Alt Text (OPTIONAL):** A string value serving as alternative text for the logo image.
+
+    - **Description (OPTIONAL):** A string value providing a description of the credential.
+
+    - **Background Color (OPTIONAL):** A string value representing the background color of the credential, specified using numerical color values defined in CSS Color Module Level 3.
+
+    - **Text Color (OPTIONAL):** A string value representing the text color of the credential, specified using numerical color values defined in CSS Color Module Level 3.
+
+## Branding in real-world
+We're using branding in this project. So in order to get familiar with it, you can visit one of our config json files in `packages/agent/conf/examples/oid4vci_metadata`. You can find the branding related information (as the spec suggests in `credentials_supported` field of the metadata)
+Here is a real world example of a credential branding:
+```json
+{
+  ...,
+  "credentials_supported": [
+    {
+      "display": [
+        {
+          "name": "Conference Attendee",
+          "description": "The DBC Conference Attendee credential is given to all visitors of the DBC conference.",
+          "background_color": "#3B6F6D",
+          "text_color": "#FFFFFF",
+          "logo": {
+            "url": "https://dutchblockchaincoalition.org/assets/images/icons/Logo-DBC.png",
+            "alt_text": "An orange block shape, with the text Dutch Blockchain Coalition next to it, portraying the logo of the Dutch Blockchain Coalition."
+          },
+          "background_image": {
+            "url": "https://i.ibb.co/CHqjxrJ/dbc-card-hig-res.png",
+            "alt_text": "Connected open cubes in blue with one orange cube as a background of the card"
+          }
+        }
+      ],
+      ...
+    }
+  ],
+  ...
+}
+```
+## Implementation
+
+Credential branding is typically specified by credential issuers in their metadata. They define how each credential they issue should be presented, including its name, logo, and visual style. Credential holders and verifiers use this branding information to ensure a consistent and recognizable display of the credential.
+
+In code, credential branding can be accessed and displayed using appropriate methods. For example, following code (in our oid4vci-demo-frontend) shows how to read and display credential branding. The `credentialBranding` object is populated with information extracted from the `credentials_supported` metadata parameter, which includes display properties and other branding-related details.
```

Comment:
Again, this part really should not be here. It is way to distracting. This documentation should be about how to setup an issuer. Not about how wallets should handle that information in code

Reply:
removed this part.